public/sass/_variables.dark.scss

```diff
@@ -31,39 +16,41 @@ $gray-2: #8e8e8e;
 $gray-3: #b3b3b3;
 $gray-4: #d8d9da;
 $gray-5: #ececec;
+$gray-6: #f4f5f8;
 $gray-7: #fbfbfb;
 
 $gray-blue: #212327;
 $input-black: #09090b;
 
-$white: #fff;
+$white: #ffffff;
 
 // Accent colors
 // -------------------------
-$blue: #33b5e5;
+$blue: #ff0000;
```

Comment:
`$blue: #ff0000` This is not blue

Reply:
Testing, see https://github.com/grafana/grafana/pull/15404#discussion_r256809029